utils/utils.ts

```diff
@@ -51,20 +55,49 @@ export function getEnvironmentRequiredVars() {
   }
 
   const logLevel = process.env.LOG_LEVEL ? process.env.LOG_LEVEL : "info";
+
   const uri = process.env.API_URL ? process.env.API_URL : "ws://127.0.0.1:9944";
+
   const userPassword = process.env.UI_USR_PWD
     ? process.env.UI_USR_PWD
     : "mangata123";
+
   const uiUri = process.env.UI_URL
     ? process.env.UI_URL
     : "https://staging.mangata.finance/";
+
   const mnemonicMetaMask = process.env.MNEMONIC_META
     ? process.env.MNEMONIC_META
     : " oh oh";
+
   const mnemonicPolkadot = process.env.MNEMONIC_POLK
     ? process.env.MNEMONIC_POLK
     : " oh oh";
 
+  const ethereumWsUrl = process.env.ETHEREUM_WEBSOCKET_URL
+    ? process.env.ETH_APP_ADDRESS
+    : "ws://localhost:8545";
```

Comment:
This guy is the bridge, right? 
in UI-test.yml, we modify depending on the environment the env. variables.  I just modified it yesterday, so you may need to update this branch with the latest main.
Do you mind adding those?

name: Set Configuration - Staging
name: Set Configuration - Develop

Reply:
I merged main but I'm not 100% sure what you mean - I can see the github actions variables to do with `name: Set Configuration` but what do I need to change in `utils`?

Reply:
ohh sorry, in utils you don't need to change anything, its in /.github/workflows/ui-test.yml where you need to place the env.Variables values.